kotlin-extensions/src/main/kotlin/kotlinx/js/JsSet.kt

```diff
@@ -2,12 +2,15 @@ package kotlinx.js
 
 @JsName("Set")
 external class JsSet<T> {
+    constructor()
```

Comment:
duplication

Reply:
I've tried to remove constructor without parameters - IntelliJ Idea stated, that there is only one constructor - with iterator, so I thought it's required...

Reply:
My bad :)